tests/integration/io/parquet/test_reads_public_data.py

```diff
@@ -174,6 +174,10 @@ def get_filesystem_from_path(path: str, **kwargs) -> fsspec.AbstractFileSystem:
         "parquet-benchmarking/s3a-mvp",
         "s3a://daft-public-data/test_fixtures/parquet-dev/mvp.parquet",
     ),
+    (
+        "parquet/test_parquet_limits_across_row_groups",
+        "s3://daft-public-data/test_fixtures/parquet-dev/tpch-issue#2730.parquet",
+    ),
```

Comment:
To my understanding, reading this file should have worked before. It just used to error if we applied a limit, which is not done by any tests over the URLs in `DAFT_CAN_READ_FILES`.

Reply:
Oh you're totally right. Forgot that this file doesn't apply a limit.

Moved the S3 tests